packages/wrangler/src/logger.ts

```diff
@@ -54,6 +55,16 @@ export class Logger {
 	columns = process.stdout.columns;
 
 	debug = (...args: unknown[]) => this.doLog("debug", args);
+	sanitizeDebug = (label: string, ...args: unknown[]) => {
```

Comment:
nit: rename this method

reasoning: `logger.debug` will be what future contributors reach for first (as a console.* method) – autocomplete will highlight the existence of this method

```suggestion
	debugWithSanitization = (label: string, ...args: unknown[]) => {
```